### ISSUE TITLE
Error loading plugin because of googleauth conflict

### ISSUE THREAD

**Issue by hieuhlc** [bskim45/fastlane-plugin-google_drive#21]
Please help me, I met this issue

`Error loading plugin 'fastlane-plugin-google_drive': Unable to activate google_drive-3.0.7, because googleauth-1.0.0 conflicts with googleauth (>= 0.5.0, < 1.0.0)`

Here is my Gemfile.lock

```GEM
  remote: https://rubygems.org/
  specs:
    CFPropertyList (3.0.3)
    addressable (2.7.0)
      public_suffix (>= 2.0.2, < 5.0)
    artifactory (3.0.15)
    atomos (0.1.3)
    aws-eventstream (1.1.1)
    aws-partitions (1.469.0)
    aws-sdk-core (3.114.3)
      aws-eventstream (~> 1, >= 1.0.2)
      aws-partitions (~> 1, >= 1.239.0)
      aws-sigv4 (~> 1.1)
      jmespath (~> 1.0)
    aws-sdk-kms (1.44.0)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sigv4 (~> 1.1)
    aws-sdk-s3 (1.96.1)
      aws-sdk-core (~> 3, >= 3.112.0)
      aws-sdk-kms (~> 1)
      aws-sigv4 (~> 1.1)
    aws-sigv4 (1.2.3)
      aws-eventstream (~> 1, >= 1.0.2)
    babosa (1.0.4)
    claide (1.0.3)
    colored (1.2)
    colored2 (3.1.2)
    commander (4.6.0)
      highline (~> 2.0.0)
    declarative (0.0.20)
    digest-crc (0.6.3)
      rake (>= 12.0.0, < 14.0.0)
    domain_name (0.5.20190701)
      unf (>= 0.0.5, < 1.0.0)
    dotenv (2.7.6)
    emoji_regex (3.2.2)
    excon (0.82.0)
    faraday (1.4.2)
      faraday-em_http (~> 1.0)
      faraday-em_synchrony (~> 1.0)
      faraday-excon (~> 1.1)
      faraday-net_http (~> 1.0)
      faraday-net_http_persistent (~> 1.1)
      multipart-post (>= 1.2, < 3)
      ruby2_keywords (>= 0.0.4)
    faraday-cookie_jar (0.0.7)
      faraday (>= 0.8.0)
      http-cookie (~> 1.0.0)
    faraday-em_http (1.0.0)
    faraday-em_synchrony (1.0.0)
    faraday-excon (1.1.0)
    faraday-net_http (1.0.1)
    faraday-net_http_persistent (1.1.0)
    faraday_middleware (1.0.0)
      faraday (~> 1.0)
    fastimage (2.2.4)
    fastlane (2.185.1)
      CFPropertyList (>= 2.3, < 4.0.0)
      addressable (>= 2.3, < 3.0.0)
      artifactory (~> 3.0)
      aws-sdk-s3 (~> 1.0)
      babosa (>= 1.0.3, < 2.0.0)
      bundler (>= 1.12.0, < 3.0.0)
      colored
      commander (~> 4.6)
      dotenv (>= 2.1.1, < 3.0.0)
      emoji_regex (>= 0.1, < 4.0)
      excon (>= 0.71.0, < 1.0.0)
      faraday (~> 1.0)
      faraday-cookie_jar (~> 0.0.6)
      faraday_middleware (~> 1.0)
      fastimage (>= 2.1.0, < 3.0.0)
      gh_inspector (>= 1.1.2, < 2.0.0)
      google-apis-androidpublisher_v3 (~> 0.1)
      google-apis-playcustomapp_v1 (~> 0.1)
      google-cloud-storage (~> 1.31)
      highline (~> 2.0)
      json (< 3.0.0)
      jwt (>= 2.1.0, < 3)
      mini_magick (>= 4.9.4, < 5.0.0)
      multipart-post (~> 2.0.0)
      naturally (~> 2.2)
      plist (>= 3.1.0, < 4.0.0)
      rubyzip (>= 2.0.0, < 3.0.0)
      security (= 0.1.3)
      simctl (~> 1.6.3)
      terminal-notifier (>= 2.0.0, < 3.0.0)
      terminal-table (>= 1.4.5, < 2.0.0)
      tty-screen (>= 0.6.3, < 1.0.0)
      tty-spinner (>= 0.8.0, < 1.0.0)
      word_wrap (~> 1.0.0)
      xcodeproj (>= 1.13.0, < 2.0.0)
      xcpretty (~> 0.3.0)
      xcpretty-travis-formatter (>= 0.0.3)
    fastlane-plugin-google_drive (0.7.0)
      google-apis-drive_v3 (>= 0.5.0, < 1.0.0)
      google-apis-sheets_v4 (>= 0.4.0, < 1.0.0)
      google_drive (~> 3, >= 3.0.7)
    gh_inspector (1.1.3)
    google-apis-androidpublisher_v3 (0.6.0)
      google-apis-core (~> 0.1)
    google-apis-core (0.3.0)
      addressable (~> 2.5, >= 2.5.1)
      googleauth (~> 0.14)
      httpclient (>= 2.8.1, < 3.0)
      mini_mime (~> 1.0)
      representable (~> 3.0)
      retriable (>= 2.0, < 4.0)
      rexml
      signet (~> 0.14)
      webrick
    google-apis-drive_v3 (0.10.0)
      google-apis-core (>= 0.3, < 2.a)
    google-apis-iamcredentials_v1 (0.4.0)
      google-apis-core (~> 0.1)
    google-apis-playcustomapp_v1 (0.3.0)
      google-apis-core (~> 0.1)
    google-apis-sheets_v4 (0.7.0)
      google-apis-core (>= 0.3, < 2.a)
    google-apis-storage_v1 (0.4.0)
      google-apis-core (~> 0.1)
    google-cloud-core (1.6.0)
      google-cloud-env (~> 1.0)
      google-cloud-errors (~> 1.0)
    google-cloud-env (1.5.0)
      faraday (>= 0.17.3, < 2.0)
    google-cloud-errors (1.1.0)
    google-cloud-storage (1.31.1)
      addressable (~> 2.5)
      digest-crc (~> 0.4)
      google-apis-iamcredentials_v1 (~> 0.1)
      google-apis-storage_v1 (~> 0.1)
      google-cloud-core (~> 1.2)
      googleauth (~> 0.9)
      mini_mime (~> 1.0)
    google_drive (3.0.7)
      google-apis-drive_v3 (>= 0.5.0, < 1.0.0)
      google-apis-sheets_v4 (>= 0.4.0, < 1.0.0)
      googleauth (>= 0.5.0, < 1.0.0)
      nokogiri (>= 1.5.3, < 2.0.0)
    googleauth (0.16.2)
      faraday (>= 0.17.3, < 2.0)
      jwt (>= 1.4, < 3.0)
      memoist (~> 0.16)
      multi_json (~> 1.11)
      os (>= 0.9, < 2.0)
      signet (~> 0.14)
    highline (2.0.3)
    http-cookie (1.0.4)
      domain_name (~> 0.5)
    httpclient (2.8.3)
    jmespath (1.4.0)
    json (2.5.1)
    jwt (2.2.3)
    memoist (0.16.2)
    mini_magick (4.11.0)
    mini_mime (1.1.0)
    multi_json (1.15.0)
    multipart-post (2.0.0)
    nanaimo (0.3.0)
    naturally (2.2.1)
    nokogiri (1.12.5-x86_64-darwin)
      racc (~> 1.4)
    os (1.1.1)
    plist (3.6.0)
    public_suffix (4.0.6)
    racc (1.5.2)
    rake (13.0.3)
    representable (3.1.1)
      declarative (< 0.1.0)
      trailblazer-option (>= 0.1.1, < 0.2.0)
      uber (< 0.2.0)
    retriable (3.1.2)
    rexml (3.2.5)
    rouge (2.0.7)
    ruby2_keywords (0.0.4)
    rubyzip (2.3.0)
    security (0.1.3)
    signet (0.15.0)
      addressable (~> 2.3)
      faraday (>= 0.17.3, < 2.0)
      jwt (>= 1.5, < 3.0)
      multi_json (~> 1.10)
    simctl (1.6.8)
      CFPropertyList
      naturally
    terminal-notifier (2.0.0)
    terminal-table (1.8.0)
      unicode-display_width (~> 1.1, >= 1.1.1)
    trailblazer-option (0.1.1)
    tty-cursor (0.7.1)
    tty-screen (0.8.1)
    tty-spinner (0.9.3)
      tty-cursor (~> 0.7)
    uber (0.1.0)
    unf (0.1.4)
      unf_ext
    unf_ext (0.0.7.7)
    unicode-display_width (1.7.0)
    webrick (1.7.0)
    word_wrap (1.0.0)
    xcodeproj (1.19.0)
      CFPropertyList (>= 2.3.3, < 4.0)
      atomos (~> 0.1.3)
      claide (>= 1.0.2, < 2.0)
      colored2 (~> 3.1)
      nanaimo (~> 0.3.0)
    xcpretty (0.3.0)
      rouge (~> 2.0.7)
    xcpretty-travis-formatter (1.0.1)
      xcpretty (~> 0.2, >= 0.0.7)

PLATFORMS
  x86_64-darwin-20

DEPENDENCIES
  fastlane
  fastlane-plugin-google_drive

BUNDLED WITH
   2.2.22
```

**Comment by jrescabias** [comment]
same problem here. cheers

**Comment by michalferko** [comment]
Same here!

**Comment by dokinkon** [comment]
Same here.

**Comment by if-alan** [comment]
Same here~

**Comment by hieuhlc** [comment]
I was able to run normally by using `bundle exec`
- Update plugins: `bundle exec fastlane update_plugins`
- Execute lane: `bundle exec fastlane [lane]`

**Comment by fadeltd** [comment]
I'm having this issue on fastlane `2.204.2`

`google_drive (~> 3, >= 3.0.7) was resolved to 3.0.7, which depends on googleauth (>= 0.5.0, < 1.0.0)`

```
Bundler could not find compatible versions for gem "fastlane-plugin-google_drive":
  In Gemfile:
    fastlane-plugin-google_drive (~> 0.7.0)

Bundler could not find compatible versions for gem "googleauth":
  In snapshot (Gemfile.lock):
    googleauth (= 1.1.0)

  In Gemfile:
    fastlane was resolved to 2.204.2, which depends on
      google-cloud-storage (~> 1.31) was resolved to 1.36.0, which depends on
        googleauth (>= 0.16.2, < 2.a)

    fastlane-plugin-google_drive (~> 0.7.0) was resolved to 0.7.0, which depends on
      google_drive (~> 3, >= 3.0.7) was resolved to 3.0.7, which depends on
        googleauth (>= 0.5.0, < 1.0.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

**Comment by rapito** [comment]
Before it was not necessary to run this in context of bundle exec for it to work, any insights to what changed that is now causing the problem to occur if bundle exec is not present?
